openslides/core/static/js/core/site.js

```diff
@@ -354,6 +360,32 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+// Options for TinyMCE editor used in various create and edit views.
+.factory('EditorOptions', [
+    'gettextCatalog',
+    'Mediafile',
```

Comment:
I want to list all mediafiles in image editor (so user can just select an uploaded png). Therefore we need a list of (filtered) Mediafiles.